### DISCUSSION TITLE
Failed to GetDriver graph overlay. Driver not supported.

### DISCUSSION THREAD

**Discussion by Yurii-Shostak** [containers/podman#18436] [Q&A]
Hi guys. Currently I'm trying to upgrade Podman from version 3.2.3 to 4.5.0 and have some errors. Current Linux Kernel version is 5.4.70. 
![image-2023-04-27-13-02-23-248](https://user-images.githubusercontent.com/45402302/235891151-2bbb4706-a870-46e6-b88b-1d52c4d60172.png)

If somebody already saw this type of errors, please help. I don't know, what dependencies are missed, so can't fix it by myself. On previous version of Podman (3.2.3) everything was fine.

**Comment by Luap99** [comment]
How was podman build? Sounds like you excluded the overlay driver from the build.

**Reply by Yurii-Shostak** [reply]
Yes, you are right. But when I delete all exclude flags there is compilation error: 
![image](https://user-images.githubusercontent.com/45402302/236701688-737f908f-1c46-4974-bfb8-7c059d0856f6.png)

**Reply by rhatdan** [reply]
You need to install a package like:
btrfs-progs-devel-6.2.2-1.fc38.x86_64

**Comment by Yurii-Shostak** [comment]
Hi. I try to use cross-compilation for my Podman image. This package (btrfs-progs-devel-6.2.2-1.fc38.x86_64) should be installed on target or on the build server (Ubuntu 16.04 in my case)?
Second question is about devmapper. 

![image](https://github.com/containers/podman/assets/45402302/50bcc8a2-a2b9-46f8-9842-facba2778ec9)

What package should be installed to avoid this iisue?
